runner/runner/main.py

```diff
@@ -308,8 +309,14 @@ def process_plugin(plugin_config: Mapping[str, Any]) -> Path:
         except subprocess.TimeoutExpired:
             monado_service_proc.kill()
             outs, errs = monado_service_proc.communicate()
-        print(outs, file=sys.stdout)
-        print(errs, file=sys.stderr)
+
+            ## Clean up leftover socket. It can only either be in $XDG_RUNTIME_DIR or /tmp
+            Path(os.environ['XDG_RUNTIME_DIR'] + "/monado_comp_ipc").unlink(missing_ok=True)
+            Path("/tmp/monado_comp_ipc").unlink(missing_ok=True)
+
+        ## Convert bytestrings to UTF-8 before printing
+        print("\nstdout:\n", outs.decode("utf-8"), file=sys.stdout)
+        print("\nstderr:\n", errs.decode("utf-8"), file=sys.stderr)
```

Comment:
Extreme nit: Converting to UTF-8 is an unnecessary, and there is no guarantee that the output will be all UTF-8. I think the best way to do this is:

```
sys.stdout.buffer.write(outs)
```